Source/WebCore/layout/formattingContexts/inline/text/TextUtil.cpp

```diff
@@ -383,13 +383,13 @@ bool TextUtil::shouldPreserveNewline(const Box& layoutBox)
 bool TextUtil::isWrappingAllowed(const RenderStyle& style)
 {
     // https://www.w3.org/TR/css-text-4/#text-wrap
-    return style.textWrap() != TextWrap::NoWrap;
+    return style.textWrapMode() != TextWrapMode::NoWrap;
```

Comment:
I really wish we had a boolean for `text-wrap-mode` instead of an enumeration for call sites like this.

And then we could get rid of this function.

Reply:
Same here, we should just use `style.autoWrap()` once it switches to use `text-wrap-mode`